File: R/tests/testthat/test_banditNarm_delta.R
context("Test banditNarm_delta")
library(hBayesDM)

test_that("Test banditNarm_delta", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(banditNarm_delta(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_banditNarm_delta.py
import pytest

from hbayesdm.models import banditNarm_delta


def test_banditNarm_delta():
    _ = banditNarm_delta(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
